ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -286,45 +299,64 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
-          className="mb-1"
-          mountOnEnter
-          unmountOnExit
-          onSelect={setActiveTab}
-        >
-          <Tab
-            eventKey="designer"
-            title="Designer"
-            disabled={hasInvalidSection}
-          >
-            <ErrorBoundary>
-              <div>
-                {pageToRender &&
-                    <ApiProvider api={previewApi}>
-                      <HtmlPageEditView htmlPage={pageToRender} readOnly={readOnly}
-                        siteHasInvalidSection={hasInvalidSection} setSiteHasInvalidSection={setHasInvalidSection}
-                        footerSection={localContent.footerSection} updateFooter={updateFooter}
-                        updatePage={page => updatePage(page, currentNavBarItem?.text)}/>
-                    </ApiProvider>}
-              </div>
-            </ErrorBoundary>
-          </Tab>
-          <Tab
-            eventKey="preview"
-            title="Preview"
-            disabled={hasInvalidSection}
+        { pageToRender &&
+          <Tabs
+            activeKey={activeTab ?? undefined}
+            className="mb-1"
+            mountOnEnter
+            unmountOnExit
+            onSelect={setActiveTab}
           >
-            <ErrorBoundary>
-              <ApiProvider api={previewApi}>
-                { pageToRender.sections.map((section: HtmlSection) =>
-                  <HtmlSectionView section={section} key={section.id}/>)
+            <Tab
+              eventKey="designer"
+              title="Designer"
+              disabled={hasInvalidSection}
+            >
+              <ErrorBoundary>
+                <div>
+                  {pageToRender &&
+                      <ApiProvider api={previewApi}>
+                        <HtmlPageEditView htmlPage={pageToRender} readOnly={readOnly}
+                          siteHasInvalidSection={hasInvalidSection} setSiteHasInvalidSection={setHasInvalidSection}
+                          footerSection={localContent.footerSection} updateFooter={updateFooter}
+                          updatePage={page => updatePage(page, currentNavBarItem?.text)}/>
+                      </ApiProvider>}
+                </div>
+              </ErrorBoundary>
+            </Tab>
+            <Tab
+              eventKey="preview"
+              title="Preview"
+              disabled={hasInvalidSection}
+            >
+              <ErrorBoundary>
+                <ApiProvider api={previewApi}>
+                  { pageToRender.sections.map((section: HtmlSection) =>
+                    <HtmlSectionView section={section} key={section.id}/>)
+                  }
+                  <SiteFooter footerSection={localContent.footerSection}/>
+                </ApiProvider>
+              </ErrorBoundary>
+            </Tab>
+          </Tabs> }
+        { currentNavBarItem?.itemType === 'EXTERNAL' && <div className="mt-2">
+          <form>
+            <label htmlFor="externalHref">External link</label>
```

Comment:
nit: some padding (`px-2` or `ps-2`) & a fixed size so it isn't huge on a big monitor would look nice.